docs/partner_program.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Tilt Partner Program
+layout: docs
+---
+
+We have a number of partnerships with teams that we meet with
+to get feedback and help prioritize things.
+
+Our attention is limited. We can only run active partnerships with a select few
+teams at a time. We want to make sure this is a good use of time for both of
+us. Here's what we look for:
+
+- A team of at least five engineers that expect to use something like Tilt for their day-to-day development
+- A microservice app where Tilt would be able to add value
+- A dev or dev-ops person who's willing to evaluate Tilt and get it setup
+
+When we kick off a partnership, we ask that you:
+
+- Check in with us bi-weekly over video chat to help with the first month or two of onboarding
```

Comment:
one of my favorite words!

bi·week·ly
/bīˈwēklē/

adjective
done, produced, or occurring every two weeks or twice a week.